core/extensions.js

```diff
@@ -64,6 +64,18 @@ Blockly.Extensions.register = function(name, initFn) {
 };
 
 /**
+ * Registers a new extension function that adds all key/value of mixinObj.
+ * @param {string} name The name of this extension.
+ * @param {!Object} mixinObj The values to mix in.
+ * @throws {Error} if the extension name is empty, the extension is already
```

Comment:
nit: did you mean "if the extension name is empty *or* the extension is already registered"?